src/progress_bar.rs

```diff
@@ -511,6 +511,30 @@ impl ProgressBar {
         }
     }
 
+    #[cfg(feature = "futures")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "futures")))]
```

Comment:
Please put the `cfg()` attributes below the docstring.

Reply:
Ah, sorry, I was just copying the `wrap_async_read` blocks, would you like me to change the above blocks to match

Reply:
Would be nice -- please do so in a different commit.